src/rl.py

```diff
@@ -767,10 +768,18 @@ def evaluate_proof(args: argparse.Namespace,
             coq_serapy.summarizeContext(coq.proof_context)
         eprint(f"Trying predictions {[action.prediction for action in actions]}",
                guard=args.verbose >= 2)
-        action_scores = evaluate_actions(coq, v_network, path,
+        if args.evaluate_baseline :
+            sorted_action = sorted(actions, key = lambda x : x.certainty)
```

Comment:
The actions come out of the predictor sorted by certainty, you don't need to re-sort them